completions/zsh/_brew_services

```diff
@@ -0,0 +1,104 @@
+#compdef brew
+#autoload
+
+__brew_installed_services() {
+  local -a services
+  # services=($(brew services list | awk '{print $1}' | tail -n+2))
+  services=(a b c)
```

Comment:
Looks like some debugging code that was left in? I'm getting odd completion behavior that looks like it's from this.

```
[~]
$ brew services list
Name  Status  User Plist
mysql stopped
[~]
$ brew services start
a  b  c
```